### PR TITLE
[TTAHUB-551] Create Script to Update Missing Duration Data

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Created a script to process a CSV and update duration on AR's.

## How to test

Create a CSV with two column's ID and DUR. Run the script importActivityReportDuration  with your CSV. The durations should be updated on the AR's listed in the CSV.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-551

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: src/tools/importDuration.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const promises = [];
+  try {
+    for await (const el of durationReports) {
+      const id = el.ID;
+      const duration = parseFloat(el.DUR) || 0;
+
+      if (duration === 0) {
+        auditLogger.info(`No duration set for report: ${id}, skipping`);
+      } else {
+        const report = await ActivityReport.findOne({ where: { id } });
```
Comment (jasalisbury) [inline src/tools/importDuration.js:30]: Can we add a condition here to make sure we are grabbing legacy reports, just to guard against a typo in the CSV list? Also maybe we want another condition making sure duration is null before setting it to a different value?

File: src/tools/ImportActivityReportDuration.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import {} from 'dotenv/config';
+import { option } from 'yargs';
+import importDuration from './importDuration';
+import { logger } from '../logger';
+
+const { argv } = option('file', {
+  alias: 'f',
+  description: 'Input .csv file',
+  type: 'string',
+}).option('region', {
```
Comment (jasalisbury) [inline src/tools/ImportActivityReportDuration.js:10]: Can we remove this option if it isn't used?

**Review by jasalisbury — APPROVED** [review]
Looks good!

File: src/tools/importDuration.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply src/tools/importDuration.js:30]: I have updated the code to check for both conditions.

File: src/tools/ImportActivityReportDuration.js (hunk 2 above)
Comment (AdamAdHocTeam) [reply src/tools/ImportActivityReportDuration.js]: I have removed the region param.

**Review by jasalisbury — APPROVED** [review]

**Pull request merged**